ONSdigital/response-operations-ui
# Respondent page shows account status as locked rather than suspended

# Motivation and Context

This change to show locked as the account status rather than suspended on the respondent page is necessary to remain consistent with the account status shown on the RU details page.

# What has changed

Add if statement on respondents.html so when respondent account status is suspended locked is shown instead.



# How to test?

Set a suspended respondent, search for them in ROps and locked should be shown as the account status.



# Links

[Trello] https://trello.com/c/Ro6ybSq9



# Screenshots (if appropriate):
